crates/bevy_picking/src/events.rs

```diff
@@ -278,7 +278,7 @@ pub struct DragDrop {
 }
 
 /// Dragging state.
-#[derive(Debug, Clone)]
+#[derive(Clone, PartialEq, Debug, Reflect)]
 pub struct DragEntry {
```

Comment:
Hmmm I don't believe `DragEntry` is a pointer event. Doesn't mean we don't want these impls for it but I *am* curious what you're doing that wanted them